src/dto/calibration-injection.dto.ts

```diff
@@ -207,9 +207,9 @@ export class CalibrationInjectionBaseDTO {
       },
     },
   )
-  @IsInRange(0, 9999999999.999, {
+  @IsInRange(-9999.99, 9999999999.999, {
     message: (args: ValidationArguments) => {
-      return `The value of [${args.value}] for [${args.property}] must be within the range of 0 and 9999999999.999 for [${KEY}].`;
+      return `The value of [${args.value}] for [${args.property}] must be within the range of -9999.990 and 9999999999.999 for [${KEY}].`;
     },
```

Comment:
Is this negative value correct?  Should it be the negative of the upper range?  Or at least to .999 instead of .990 ?

Reply:
updated.